README.md

```diff
@@ -237,7 +232,3 @@ VMSTE_L1PHIA4n2 input=> VMR_L1PHIA.vmstuboutPHIA4n2  output=> TE_L1PHIA4_L2PHIA3
 
 e.g. The VM memory "4" in coarse phi region "A" VMSTE_L1PHIA4 is written by VM router algo step VMR_L1PHIA (naming convention given above under HourGlassConfig.py). Here "n2" in memory name indicates that this is the second copy of the memory (where multiple copies used to avoid conflicts). The I/O pins of the proc blocks are ".vmstuboutphiA4n2" & ".innervmstubin", whose names include "in" or "out" to distinguish read or write, and also include parts of the names of the memories they connect to, so make clear what sort of data the internal logic of the proc block must read/write to the pin. (These pin names appear in the HLS code interface).
 
-## Python version and other dependencies
```

Comment:
is the code now compatible with python 3?

Reply:
It is definitely not compatible with Python 3 right now (these scripts don't run on my laptop because Python 3 is all I have installed).

Reply:
ok, so maybe that part of the docs should be re-installed/that edit reverted? @tomalin

Reply:
Agreed. Python3 section restored to README.md. (I put it immediately in front of the "Technical Details" section).

Reply:
ok great. I think this can be merged. Re python 3, is it the root dependency? It might be worth just stripping that out/rewriting it in python w/o root, if the only thing it's used for is making the wiring diagrams.

Reply:
The scripts indeed use ROOT only to make the pretty picture showing how the modules & memories are wired together in the chain.

To support python 3, we'd have to change a number of lines in our scripts. e.g. Replacing:
print "a message"
with:
print("a message")
I don't know if we could make them compatible with both python 2 & 3.

Reply:
I can take a stab at this in some point in the future.

Reply:
great. merging.